samples/word/06-lists/insert-list.yaml

```diff
@@ -1,3 +1,4 @@
+order: 1
 id: word-lists-insert-list
 name: List creation
```

Comment:
Consider changing "List creation" to "Create a list" in snippet name (for consistency with how other snippet names are phrased)